<launch>
  <arg name="team" default="1"/>
  
  <node pkg="player" name="play_node" type="play_node" output="screen">
  	<param name="team" value="$( arg team )" />
  </node>
   
</launch>
